config/opal_configure_options.m4

```diff
@@ -187,10 +187,19 @@ AC_DEFINE_UNQUOTED(OPAL_ENABLE_TIMING, $WANT_TIMING,
 AM_CONDITIONAL([OPAL_COMPILE_TIMING], [test "$WANT_TIMING" = "1"])
 AM_CONDITIONAL([OPAL_INSTALL_TIMING_BINARIES], [test "$WANT_TIMING" = "1" && test "$enable_binaries" != "no"])
 
+# Later calls to AC_USE_SYSTEM_EXTENSIONS can
```

Comment:
I don't think this is the right solution to the problem.  Let's not add random crap to CFLAGS.  Instead, let's decide what we want the behavior to be.  If we need to move up adding -g to before we call AC_USE_SYSTEM_EXTENSIONS, then let's do that.  But this is not the right fix.

Reply:
I considered that route, but there is the `$enable_debug_flags` variable, and if that is false, it will still fail in that case.